supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-data-grid.adoc

```diff
@@ -0,0 +1,102 @@
+[[red-hat-data-grid-conventions]]
+
+For documentation questions, contact jdg-docs@redhat.com.
+
+[discrete]
+[[cache-manager]]
+==== image:images/yes.png[yes] Cache Manager (noun)
+*Description*: Interface for creating caches and managing cache lifecycles. Always spell as two words with capital letters when referring to the abstract notion of a Cache Manager. When referring to specific interfaces, use `CacheManager`, `EmbeddedCacheManager`, or `RemoteCacheManager`.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[data-grid]]
+==== image:images/yes.png[yes] Data Grid (noun)
+*Description*: The short product name for Red Hat Data Grid. Red Hat Data Grid is a high-performance, distributed in-memory data store.
```

Comment:
Is there any guidance for using this only after introducing the full name first that we should include here?

An example of an entry that does this now: https://redhat-documentation.github.io/supplementary-style-guide/#red-hat-cloudforms